src/modal/modal.js

```diff
@@ -309,7 +309,9 @@ angular.module('ui.bootstrap.modal', ['ui.bootstrap.stackedMap'])
         if (elementToReceiveFocus && elementToReceiveFocus.focus) {
           elementToReceiveFocus.focus();
         } else {
-          appendToElement.focus();
+          if(appendToElement.focus) {
```

Comment:
ah yes, good point indeed. completely overlooked that after console.log debugging i guess